src/CentreonRemote/Domain/Exporter/ConfigurationExporter.php

```diff
@@ -80,8 +80,10 @@ public function import(ExportManifest $manifest): void
 
             $import = $manifest->get("import");
             foreach ($import['data'] as $data) {
-                // truncate table
-                $db->query("TRUNCATE TABLE `" . $data['table'] . "`;");
+                // empty table
+                $db->query("DELETE FROM `" . $data['table'] . "`");
+                // optimize table
+                $db->query("ALTER TABLE `" . $data['table'] . "`");
```

Comment:
will $data['table'] be able to contain something like "MODIFY", "ADD COLUMN" or some other like queries needing a "IF NOT EXIST" if being already added to the table ?
If so, it won't work on deported MySQL 8 DB